docs/_documentations/remotedeploy-combo.md

```diff
@@ -10,59 +10,64 @@ parent: installoncloud
 
 `cwctl --insecure install remote` 
 
-and requires various flags to specify where and what to install.
+This command requires various flags to specify where and what to install.
 
-### Deploying Codewind with CWCTL
+## Deploying Codewind with CWCTL
 
-1.  Open a new terminal window on your local workstation
-2.  Navigate to your home directory and to the Codewind cli:
+1.  Open a new terminal window on your local workstation.
+2.  Navigate to your home directory, and then to the Codewind CLI:
 
 ```
 cd ~/.codewind/0.8.0
 ```
 
-Ensure you are logged into your Kubernetes or Openshift cluster
+Ensure that you are logged into your Kubernetes or Openshift cluster:
 
 ```
 $ kubectl get namespaces
 ```
 
-If the command is successful you should see a list of current namespaces.  If not,  ensure you are logged into your Kubernetes or Openshift cluster.
+If the command is successful, you see a list of current namespaces. If not, ensure that you are logged into your Kubernetes or Openshift cluster.
 
-### Determine your Cloud ingress domain
+## Determine your Cloud ingress domain
 
-The cli command requires an ingress domain,  you can find yours based on any of the existing routes:
+The CLI command requires an ingress domain. You can find your ingress domain based on any of the existing routes:
 
 ```
 oc get routes -n default
 NAME               HOST/PORT                                                                                                          PATH      SERVICES           PORT               TERMINATION   WILDCARD
 registry-console   registry-console-default.mycluster-12345-7674b4bd9abbdeea5be228236d5275c9-0001.eu-gb.containers.appdomain.cloud             registry-console   registry-console   passthrough   None
 ```
 
-In the above example the ingress domain needed is:
+In the above example the ingress domain is:
 
 ```
 mycluster-12345-7674b4bd9abbdeea5be228236d5275c9-0001.eu-gb.containers.appdomain.cloud
 ```
 
-### Running the cli command
+## Running the CLI command
 
-Determine the following for your cloud deployment
+Determine the following for your cloud deployment:
 
-* {namespace}: cwctl will create the namespace if it does not yet exist
-* {kadminuser} & {kadminpass}:  Initial Keycloak administrator username and password
-* {kdevuser} & {kdevpass}: A developer username and password that will be granted access to this deployment of Codewind. cwctl will create the user and add it to the realm if it does not exist.
-* {ingress}: the ingress domain for your cloud environment
+- {namespace}: The `cwctl` command creates the namespace if it does not yet exist.
+- {kadminuser} & {kadminpass}: Initial Keycloak administrator username and password.
+- {kdevuser} & {kdevpass}: A developer username and password that will be granted access to this deployment of Codewind. The `cwctl` command creates the user and adds it to the realm if it does not exist.
+- {ingress}: The ingress domain for your cloud environment.
```

Comment:
- Should these values be in code phrase format, such as `{kdevuser}`?
- Unless there's a specific reason we need the `&` character, it would be good to go with "and" instead.